puppet/zulip/manifests/app_frontend_once.pp

```diff
@@ -36,6 +36,11 @@
   }
 
   # Daily
+  zulip::cron { 'update-stream-active-status':
+    hour   => '1',
+    minute => '0',
```

Comment:
Let's not add more things at minute 0. We want them spread out, both for debugging reasons and load reasons. 40 seems OK.

Reply:
Changed to `40`.